orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -329,9 +359,12 @@ function logAjaxError(e){
     console.log("responseText: " + e.responseText);*/
 }
 
-function getBaseUri() {
-    return 'https:' == document.location.protocol ? orcidVar.baseUri
-            : orcidVar.baseUriHttp;
+function getBaseUri() {    
+    var uri = location.protocol + '//' + location.host
```

Comment:
Just for the record. 
This ideally will be moved to be provided on a service in the future. 

This also applies for most function on script.js